lib/creator/transformations/module/module.js

```diff
@@ -0,0 +1,164 @@
+															'init',
+															j.identifier(key),
+															j.literal(
+																subProps[key].__paths[0].value.program.body[0].expression.value
+															)
+														)
+													);
+												} else {
+													moduleProp.value.elements[seen].properties.push(
+														j.property(
+															'init',
+															j.identifier(key),
+															j.literal(
+																subProps[key]
+															)
+														)
+													);
+												}
+											}
+											if(key === 'enforce') {
+												moduleProp.value.elements[seen].properties.push(
+													j.property('init', j.identifier(key), j.literal(subProps[key]))
+												);
+											}
+											if(key === 'loader') {
+												moduleProp.value.elements[seen].properties.push(
+													j.property('init', j.identifier(key), j.literal(subProps[key]))
+												);
+											}
+											if(key === 'options') {
+												let optionVal = j.property('init', j.identifier(key), j.identifier('presets'));
+												optionVal.value.type = 'ObjectExpression';
+												optionVal.value.properties = [];
+
+												Object.keys(subProps[key]).forEach( (optionProperty) => {
+													if(Array.isArray(subProps[key][optionProperty])) {
+														const optionArray = j.property(
+															'init',
+															j.identifier(optionProperty),
+															j.arrayExpression([])
+														);
+														subProps[key][optionProperty].forEach( (n) => {
+															optionArray.value.elements.push(j.literal(n));
+														});
+														optionVal.value.properties.push(optionArray);
+													} else {
+														optionVal.value.properties.push(
+															j.property(
+																'init',
+																j.identifier(optionProperty),
+																j.literal(key[optionProperty])
+															)
+														);
+													}
+												});
+												moduleProp.value.elements[seen].properties.push(optionVal);
+											}
+											if(key === 'use') {
+												let UseVal = j.property('init', j.identifier(key), j.arrayExpression([]));
+
+												Object.keys(subProps[key]).forEach( (optionProperty) => {
+													if(typeof(subProps[key][optionProperty]) === 'string') {
+														UseVal.value.elements.push(j.literal(subProps[key][optionProperty]));
+													} else {
+														let loaderProperty = j.objectExpression([]);
+
+														Object.keys(subProps[key][optionProperty]).forEach( subOptionProp => {
+															if(typeof(subProps[key][optionProperty][subOptionProp]) === 'string') {
+																loaderProperty.properties.push(
+																	j.property('init',
+																	j.identifier(subOptionProp),
+																	j.literal(subProps[key][optionProperty][subOptionProp])
+																)
+															);
+															} else {
+																let subSubProps = j.objectExpression([]);
```

Comment:
Sorry but this is not possible to read. Line lengths are way too over the eslint limit and it should be split into smaller functions I think.